test/integration/kueuectl/list_test.go

```diff
@@ -134,44 +132,48 @@ very-long-local-queue-name   cq1                            0
 		})
 
 		// Simple client set that are using on unit tests not allow to filter by field selector.
-		ginkgo.It("Should print local queues list filtered by field selector", func() {
-			streams, _, output, errOutput := genericiooptions.NewTestIOStreams()
-			configFlags := CreateConfigFlagsWithRestConfig(cfg, streams)
-			executeTime := time.Now()
-			kueuectl := app.NewKueuectlCmd(app.KueuectlOptions{ConfigFlags: configFlags, IOStreams: streams, Clock: testingclock.NewFakeClock(executeTime)})
+		ginkgo.It("Should print cluster queues list filtered by field selector", func() {
```

Comment:
This is this test changed since the title says "print cluster queues list with paging"? Is it also failing for the same reason?

Reply:
I can't see the problem with this test. But to be sure I added eventually here too.

Reply:
I suggest not add it proactively, unless we have some good reason to believe it is flaky.